src/telemetry/telemetryProps.ts

```diff
@@ -4,8 +4,17 @@
 *--------------------------------------------------------------------------------------------*/
```

Comment:
Are you planning to change `telemetryProps` to be more specific ie naming it `DeployImageApiTelemetryProps` or something?

Reply:
I was going to rename the file to something like `commandTelemetryProps` and then continue sticking all the command props in this file.  The shared props can be contained in separate files

Reply:
Unless you mean in the command itself, which I'm actually doing this:

![image](https://github.com/microsoft/vscode-azurecontainerapps/assets/40250218/9ce39803-743e-42d2-8945-f192efdfc246)

Reply:
Unless you're referring to how it's being called, because I'm actually renaming all of them to `TelemetryProps` when imported because it's easier to read:

![image](https://github.com/microsoft/vscode-azurecontainerapps/assets/40250218/03cd554e-780a-4b3b-87bf-cb6215442c96)

Reply:
No, I was referring to the name of the file.